continuous_integration/azure/install.bat

```diff
@@ -27,6 +27,9 @@ if %USE_CONDA% == yes (
 ::    conda install -q -y python=3.6.8=h9f7ef89_0
 ::)
 
+:: Install diff-match-patch
+pip install -q diff-match-patch
```

Comment:
Please remove this from here and instead add `diff-match-patch` to `requirements/conda.txt`. Also add a mention to this package in our Readme.